src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline task that stores the description of the task
+ * as well as a date deadline that gets reformatted.
+ */
+public class Deadline extends Task {
+
+    protected LocalDate by;
+
+    /**
+     * Creates a Deadline Task
+     *
+     * @param description in String format.
+     * @param by          Which is a LocalDate.
```

Comment:
Is there perhaps some additional space indented error or there was something here but I can't view it?
```suggestion
     * @param by Which is a LocalDate.
```